### PR TITLE
Adopt flutter lints

### PR DESCRIPTION
**Pull request by domesticmouse**

### CODE REVIEW

File: adaptive_navigation/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# [0.0.4] - Ju;ly 14, 2021
```
Comment (guidezpl) [inline adaptive_navigation/CHANGELOG.md]: nit: typo

**Review by guidezpl — APPROVED** [review]
Awesome, thank you for this PR!

Tiny nits

File: adaptive_breakpoints/analysis_options.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+include: package:flutter_lints/flutter.yaml
+
+linter:
+  rules:
+    prefer_single_quotes: true
```
Comment (guidezpl) [inline adaptive_breakpoints/analysis_options.yaml]: nit: EOF line

File: adaptive_navigation/CHANGELOG.md (hunk 1 above)
Comment (domesticmouse) [reply adaptive_navigation/CHANGELOG.md]: done

File: adaptive_breakpoints/analysis_options.yaml (hunk 2 above)
Comment (domesticmouse) [reply adaptive_breakpoints/analysis_options.yaml]: done

**Comment by domesticmouse** [comment]
thanks!

**Pull request merged**